### DISCUSSION TITLE
Degrade batteries slower (or not at all) when switched off.

### DISCUSSION THREAD

**Discussion by Ek-Videogames** [FakeFishGames/Barotrauma#14739] [Ideas and suggestions]
Junctions and various power consuming devices degrade less when switched off, or when not 'under stress' (aka powered but not being actively used), but batteries degrade at full speed at all times. This is particularly problematic with things like battery powered drones or shuttles, or submarines with backup power batteries that can be switched off.

- The strategy of turning of submarine's power off when the crew leaves to investigate a wreck or alien ruin, only to come back and everything on the submarine is fried. 
- Or a drone like the Kastrull/Remora or any custom submarine, needing constant upkeep to keep the battery condition topped up and ready because they're toast after 2 minutes even when the drone isn't even being used.

It would be nice if batteries' (and supercaps maybe?) degradation speed took into account how much power they are inputting/outputting as their measurement of "stress" (voltage), and/or using the new 'disable output' connection to disable the battery should also essentially do the same thing as having the battery under "0% stress"

**Comment by jigardi** [comment]
Huh... I've never noticed this - but then again, I always store batteries in the charger.  Maybe this is why I do that so obsessively.
